presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/testing/SqlInvokedFunctionTestUtils.java

```diff
@@ -44,7 +44,7 @@ private SqlInvokedFunctionTestUtils()
             parseTypeSignature(DOUBLE),
             "power tower",
             RoutineCharacteristics.builder().setDeterminism(DETERMINISTIC).build(),
-            "pow(x, x)",
+            "RETURN pow(x, x)",
```

Comment:
This would cause compatibility issue when roll out to production. Make sure that's handled during deployment.